### PR TITLE
add/promoted posts list page

### PR DESCRIPTION
**Pull request by benzhu56**
#### Proposed Changes
**update**:
* scalfold "Promoted posts" page with two tabs: posts and campaigns. (needs more styling and functionality)
* my home page CTA buttons: swipeable, quick links, learning center now all take us to this page.

calypso-promoted-posts

#### Testing Instructions

*

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have you sent any new strings for translation (PCYsg-1vr-p2) ASAP?

Related to #

### CODE REVIEW

**Comment by benzhu56** [comment]
Update: got loading state working, implemented (low fidelity as Figma is just a screenshot) posts list and campaigns list. 
The PR is getting bigger, I would like to merge it before adding more details in.
@jjolmo do you mind having a review? Thanks!
update

File: client/my-sites/promote-post/components/campaign-item/index.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+				</div>
+			</div>
+		</div>
+	);
+	return (
+		<FoldableCard header={ header }>
+			<div className="campaign-item__stats">
+				<div className="campaign-item__stats-row1">
+					<div className="campaign-item__reach">
+						<div className="campaign-item__reach-value">123 todo</div>
```
Comment (jjolmo) [inline client/my-sites/promote-post/components/campaign-item/index.tsx:30]: sooner than later we will wire this to a new method to get the report of the campaign

File: client/my-sites/promote-post/components/campaigns-empty/index.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export default function CampaignsEmpty() {
+	return (
+		<div className="campaigns-empty__container">
+			<EmptyContent
+				className="campaigns-empty"
+				title={ translate( 'Looking for a campaign?' ) }
+				line={ 'There are no running campaigns now.' }
+				action={ 'Start a campaign' }
+				actionURL={ 'todo' }
+				secondaryAction={ 'Learn more' }
```
Comment (jjolmo) [inline client/my-sites/promote-post/components/campaigns-empty/index.tsx:15]: I guess we miss "translate" here, but this feature might have design changes so I think it's ok to postpone them

**Review by jjolmo — APPROVED** [review]
Works in my local (tm). Great work!

**Review by gravityrail — APPROVED** [review]
Works for me. I'm going to merge and deploy.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7469437

Thank you @benzhu56 for including a screenshot in the description! This is really helpful for our translators.

File: client/my-sites/promote-post/components/campaign-item/index.tsx (hunk 1 above)
Comment (benzhu56) [reply client/my-sites/promote-post/components/campaign-item/index.tsx:30]: cool, I'll NOT touch this for now until we get more certainty.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7469437

Thank you @benzhu56 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7469437).

**Pull request merged**